.github/workflows/update-prerun-script.yml

```diff
@@ -0,0 +1,86 @@
+name: Update pre build script on Saucelabs
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+#  schedule:
+#    - cron: '0 0 1 */2 *'
```

Comment:
delete?

Reply:
I need to change it to this cron once I test my changes after the code is merged in
Currently I am going to run it every 10 mins just to check if my code works. After that it will run every two months